ImGuiStyler/Alignment.cs

```diff
@@ -4,14 +4,29 @@ namespace ktsu.ImGuiStyler;
 using ImGuiNET;
 using ktsu.ScopedAction;
 
+/// <summary>
+/// Provides methods for aligning content within a container.
+/// </summary>
 public static class Alignment
 {
+	/// <summary>
+	/// Centers the content within the available content region.
```

Comment:
Should this mention that the cursor will be advanced to the next line?

Reply:
I dont think its super nescessary as advancing to the next line is the default imgui behavior